node/src/tasks/bsp_upload_file.rs

```diff
@@ -404,7 +385,7 @@ where
         Ok(())
     }
 
-    async fn unvolunteer_file(&self, file_key: HasherOutT<T>) -> anyhow::Result<()> {
+    async fn unvolunteer_file(&self, file_key: H256) -> anyhow::Result<()> {
```

Comment:
How about using `HasherOutT<StorageProofsMerkleTrieLayout>` in this document, instead of `H256`?